.github/workflows/Basic.yml

```diff
@@ -72,8 +71,8 @@ jobs:
       # TODO: we should check
       # CHANGES_IN_REPO=$(git status --porcelain)
       # after this, but I don't know how
-      - name: Generate Schema
-        uses: actions-rs/cargo@v1
-        with:
-          command: schema
-          args: --locked
+      # - name: Generate Schema
```

Comment:
Can this be removed instead of just commented out?

Reply:
Fix the issues as instructed.